playbooks/luna-devel.yml

```diff
@@ -0,0 +1,29 @@
+- hosts: devel
+  become: true
+  vars:
+    katello_repositories_version: nightly
+    foreman_installer_scenario: katello-devel
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--katello-devel-github-username {{ katello_devel_github_username }}"
+      - "--katello-devel-enable-ostree=true"
+      - "--katello-devel-extra-plugins theforeman/foreman_remote_execution"
```

Comment:
Does this set up the proxies required for these features?

Reply:
no, I think it would be useful to add installer options so that proxy is installed with all respective plugins via rpms until we have proper dev setup of proxy with plugins too

Reply:
The plugin / repository was renamed from "theforeman/foreman-docker" to "theforeman/foreman_docker".
https://github.com/theforeman/foreman_docker

Reply:
Sorry this is comment to line 18.